packages/pkg-tests/pkg-tests-core/sources/utils/fs.js

```diff
@@ -173,3 +173,15 @@ exports.readJson = async function readJson(source: string): Promise<any> {
     throw new Error(`Invalid json file (${source})`);
   }
 };
+
+exports.chmod = function chmod(target: string, mod: number): Promise<void> {
+  return fs.chmod(target, mod);
```

Comment:
Why is the return type for this, is a promise?

Reply:
```
const fs = require('fs-extra');
```

fs-extra exposes the fs functions but make them promise-ready.